cheroot/ssl/__init__.py

```diff
@@ -1,5 +1,6 @@
 """Implementation of the SSL adapter base interface."""
 
+import socket
```

Comment:
Let's keep it from polluting the importable namespace.
```suggestion
import socket as _socket
```